### PR TITLE
Additional environment helper scripts

### LINKED ISSUE #162: Why does thisdd4hep try to set up env for external libraries?
In my opinion, this should not happen.  On my system, it also fails to set up properly unless I remove the external library setup lines, making it even more annoying.

### PR DESCRIPTION
**Pull request by decibelCooper**
This request resolves #162 in the manner suggested in the thread.

BEGINRELEASENOTES

- Added environment helper scripts `thisdd4hep_only.(c)sh` that only set up variables for DD4hep and not for dependencies.

ENDRELEASENOTES

### CODE REVIEW

**Comment by andresailer** [comment]
You also have to `configure` the scripts like here:
https://github.com/AIDASoft/DD4hep/blob/master/CMakeLists.txt#L151
And install them like a few lines below that line

**Comment by decibelCooper** [comment]
@andresailer My apologies for the haste.  This has been updated and tested.

**Comment by andresailer** [comment]
No worries, thanks for your contribution

File: cmake/thisdd4hep_only.sh [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+# Author: Pere Mato. F. Gaede, M.Frank
+#-------------------------------------------------------------------------------
+#
+#echo " ### thisdd4hep.sh:   initialize the environment for DD4hep ! " 
+#
+#-----------------------------------------------------------------------------
+dd4hep_parse_this()   {
+    package=${2};
+    if [ "x${1}" = "x" ]; then
+	if [ ! -f bin/this${package}.sh ]; then
```
Comment (andresailer) [inline cmake/thisdd4hep_only.sh]: This should be `this${package}_only.sh`

File: cmake/thisdd4hep_only.sh [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+            echo ERROR: must "cd where/${package}/is" before calling ". bin/this${package}.sh" for this version of bash!;
```
Comment (andresailer) [inline cmake/thisdd4hep_only.sh]: This should be `this${package}_only.sh`

**Pull request merged**